x-pack/plugins/secops/public/components/timeline/body/renderers/auditd_loggedin_row_renderer.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { get } from 'lodash/fp';
+import React from 'react';
+import styled from 'styled-components';
+
+import { RowRenderer, RowRendererContainer } from '.';
+import { AuditdLoggedinDetails } from './auditd_loggedin_details';
+
+const AuditLoggedinRow = styled.div`
+  width: 100%;
+  overflow: hidden;
+  &:hover {
+    border: 1px solid ${props => props.theme.eui.euiColorMediumShade};
```

Comment:
I've been migrating this style to:
`background-color: ${props => props.theme.eui.euiTableHoverColor};`
I might make a common style for it, but you can pre-migrate to this style now, if you want. It makes it consistent with the rest of the hover styles, and avoids the "bouncing" effect caused by the border.

Reply:
Updated to those colors now.